### PR TITLE
Front of register is done

### PR DESCRIPTION
**Pull request by SoloNoah**
closes issue #31

### CODE REVIEW

**Comment by AvielCo** [comment]
im checking, please hold your horses.

**Comment by SoloNoah** [comment]
time is money! vamanos mi amigo!

**Review by AvielCo — CHANGES_REQUESTED** [review]
Handle my Register.js review please :)

File: client/src/pages/Register.js [hunk 1]
```diff
@@ -1,15 +1,36 @@
...
+
+import "./Register.css";
 
 function Register() {
+  const [inputUsername, setUsername] = useState("");
+  const [inputPassword, setPassword] = useState("");
+  const [inputEmail, setEmail] = useState("");
+  const [inputName, setName] = useState("");
+
   const handleSubmit = (event) => {
```
[the remark this answers is not in the document]
Comment (SoloNoah) [reply client/src/pages/Register.js:16]: I'm on it!

[the remark this answers is not in the document]
Comment (AvielCo) [reply client/src/pages/Register.js:16]: thanks <3 dont forget to use .env secret key

**Pull request merged**